src/main/java/file/FileManager.java

```diff
@@ -0,0 +1,110 @@
+package file;
+
+import task.*;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class FileManager {
+
+    private static Path taskDataPath = Paths.get(System.getProperty("user.dir"), "data", "tasks.txt");
+
+    private static Path dataDirPath = Paths.get(System.getProperty("user.dir"), "data");
+
+    public static void writeToFile(Task task) throws IOException {
+        FileWriter fw = new FileWriter(taskDataPath.toString(), true);
+        String textToAdd = taskToText(task);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+
+    public static void startReading() {
+        try {
+            makeMissingDirectory();
+            readFile();
+        } catch (FileNotFoundException e) {
+            System.out.println("File cannot be found");
+        } catch (IOException e) {
+            System.out.println("Some IO error has occurred");
+        }
+    }
+
+    private static void readFile() throws FileNotFoundException {
+        String filePath = taskDataPath.toString();
+        File f = new File(filePath);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            String[] input = s.nextLine().split(" | ");
```

Comment:
Similarly, consider setting the delimiter you use as a constant, in order to eliminate magic literals.